microsoft/atlas-design
# Update warning color

Link: [preview](http://localhost:1111/)

This PR updates the warning color to match Fluent on light theme. Fluent uses orange colors for warning, but they also have yellow colors. Therefore, this PR adds both a yellow and orange palette. Also updating warning colors for dark and high contrast theme to make everything orange for consistency. 

## Testing

1. Visit the following pages to review the affected colors, should still work on dark and high contrast theme. Compare with https://react.fluentui.dev/ . Also compare with https://design.learn.microsoft.com/ for the current Atlas colors.

- [Badge | Atlas Design | Microsoft](http://localhost:1111/components/badge.html)
  - Fluent storybook shows warning badge as yellow, which seems incorrect, https://react.fluentui.dev/?path=/docs/components-badge-badge--docs
  - Figma shows orange, https://www.figma.com/design/FO4ImdAytGCc5DQXpNFObI/Microsoft-Fluent-2-Web-(Community)?node-id=8911-3186&node-type=canvas&t=LXqfwjT3ovisb3kn-0
- [Button | Atlas Design | Microsoft](http://localhost:1111/components/button.html)
  - Fluent doesn't have warning buttons
- [Notification | Atlas Design | Microsoft](http://localhost:1111/components/notification.html)
    - Inspect the last notification and change `notification-primary` to `notification-warning`. 
    - https://react.fluentui.dev/?path=/docs/components-messagebar--docs
- [Progress Bar | Atlas Design | Microsoft](http://localhost:1111/components/progress-bar.html)
    - Fluent uses a dark orange instead, which seems inconsistent. Keeping as warning base for consistency
    - https://react.fluentui.dev/?path=/docs/components-progressbar--docs 

## Additional information

[Optional]

## Contributor checklist
